src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write your code here
+const numbers = [...document.querySelectorAll('.population')];
+
+let avarage = 0;
+let total = 0;
+
+numbers.forEach((el) => {
+  const newData = el.textContent.replaceAll(',', '');
```

Comment:
Consider using 'replace(/,/g, '')' instead of 'replaceAll' for better compatibility across different environments.